app/Services/GenerateLateFeeService.php

```diff
@@ -69,14 +69,15 @@ public function handle($day)
             # code...
             $data = [
                 'order_id' => $item->id,
-                'amount_due' => $this->paystackService->getLateFee($item),
+                'amount_due' => $this->paystackService->getLateFee($item) === 'invalid' ? 0 : $this->paystackService->getLateFee($item),
```

Comment:
we don't need to generste the record at all instead of zero

Reply:
done